### PR TITLE
Update unitofmeasures.csv

### PR DESCRIPTION
**Pull request by philstubbington**
Description missing for last two UOM.

### CODE REVIEW

File: datamodel/referencedata.d/unitofmeasures.csv [hunk 1]
```diff
@@ -7,5 +7,5 @@ CEL,Celsius
 FAH,Fahrenheit
 FOT,Foot
 MTR,Meter
-MQH,
-FQH,
+MQH,Cubic Meter Per Hour
+FQH,Cubic Feet Per Hour
```
Comment (HenrikHL) [inline datamodel/referencedata.d/unitofmeasures.csv:11]:
```suggestion
FQH,Cubic Foot Per Hour
```

**Review by HenrikHL — APPROVED** [review]

**Comment by HenrikHL** [comment]
@philstubbington - FYI I have asked UNECE to change their abbreviation for **Cubic Foot Per Hour** from `2K` (which is their current proposal) to FQH (which is equivalent to the value in the file you modified)

**Review by HenrikHL — APPROVED** [review]

**Comment by nt-gt** [comment]
For some reason the CI job is not being run for this PR and I cannot see anything on the DCSA end to affect this.

@philstubbington: Do you see anything in your end that would stop the CI from being run? I think it will be run in your fork, so maybe you need to allow CIs to run for that?

Alternatively, we can copy the branch into the DCSA repo and open a new PR from the DCSA side, which might side-step the issue.

**Comment by nt-gt** [comment]
Maybe we are missing a PR trigger.

**Comment by nt-gt** [comment]
Thanks for your contribution to DCSA. :)

**Pull request merged**